namespaces/exec.go

```diff
@@ -327,7 +327,7 @@ func DefaultSetupCommand(container *libcontainer.Config, console, dataPath, init
 func SetupCgroups(container *libcontainer.Config, nspid int) (map[string]string, error) {
 	if container.Cgroups != nil {
 		c := container.Cgroups
-		if systemd.UseSystemd() {
+		if container.Cgroups.Driver == "systemd" && systemd.UseSystemd() {
```

Comment:
We should remove the ` && systemd.UseSystemd()` from this and the one below like you did on the one above